_includes/manuals/1.8/5.2.10_cr_encryption_key.md

```diff
@@ -0,0 +1,9 @@
+Compute resource passwords and secrets are stored on the Foreman database using a secret - the encryption key - and ciphered using AES-256-CBC.
+The encryption key can usually be found in config/initializers/encryption_key.rb. The value of the ENCRYPTION_KEY variable must be at least 32 bytes long.
```

Comment:
On a package installation, this is a symlink to a file that's actually in /etc/foreman.  I wonder if it's worth saying that /etc is the location, and as a sidenote say it's symlinked to config.